StripeConnect/StripeConnectTests/Internal/Webview/MessageHandlers/AccountSessionClaimedMessageHandlerTests.swift

```diff
@@ -9,19 +9,17 @@
 import XCTest
 
 class AccountSessionClaimedMessageHandlerTests: ScriptWebTestBase {
-    func testMessageSend() {
-        let expectation = self.expectation(description: "Message received")
```

Comment:
expectation is no longer needed because `evaluateAccountSessionClaimed` was changed to an async method which we now await